.github/workflows/ci-master.yml

```diff
@@ -32,18 +32,15 @@ jobs:
       - name: Build
         run: pnpm build
 
-      - name: Test
-        run: pnpm test
-
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3
+      - name: Cache build artifacts
+        uses: actions/cache@v3
         with:
-          files: packages/@n8n/client-oauth2/coverage/cobertura-coverage.xml,packages/cli/coverage/cobertura-coverage.xml,packages/core/coverage/cobertura-coverage.xml,packages/design-system/coverage/cobertura-coverage.xml,packages/editor-ui/coverage/cobertura-coverage.xml,packages/nodes-base/coverage/cobertura-coverage.xml,packages/workflow/coverage/cobertura-coverage.xml
-
-      - name: Lint
-        env:
-          CI_LINT_MASTER: true
-        run: pnpm lint
+          path: |
+            /github/home/.cache
+            /github/home/.pnpm-store
+            ./node_modules
+            ./packages
+          key: ${{ github.sha }}-base:18-test-lint
 
       - name: Notify Slack on failure
```

Comment:
Do we need to duplicate the Slack notification step or will it capture issues from parallel jobs?